common/src/main/java/com/revenuecat/purchases/common/errors.kt

```diff
@@ -56,7 +56,7 @@ fun Exception.toPurchasesError(): PurchasesError {
 private fun BackendErrorCode.toPurchasesError(underlyingErrorMessage: String) =
     PurchasesError(this.toPurchasesErrorCode(), underlyingErrorMessage)
 
-internal fun HTTPResult.toPurchasesError(): PurchasesError {
+fun HTTPResult.toPurchasesError(): PurchasesError {
```

Comment:
this is now officially a new API right? is it intentional? If so, we need to have docs for it, and API tests

Reply:
ohh I guess it's not really public because it's not in the public module. Can't wait until we revamp modules for android tbh.